src/Commands/EnvDeleteCommand.php

```diff
@@ -204,11 +185,11 @@ function ($item) {
         }
 
         // Delete each of the selected environments.
-        foreach ($oldestEnvironments as $env_id) {
+        foreach ($environmentsToDelete as $env_id) {
             $site_env_id = "{$site_id}.{$env_id}";
```

Comment:
I don't think `$site_id` is defined for this function. When I run this command locally I get a notice about an undefined variable and then a Terminus error about giving the environment argument properly.